File: in_Java/0908 Smallest Range I.java
class Solution {
    public int smallestRangeI(int[] A, int K) {
        int mmin = A[0];
        int mmax = A[0];
        for(int i : A){
            mmin = Math.min(mmin,i);
            mmax = Math.max(mmax,i);
        }
        if(mmax - mmin > K+K) return mmax - mmin - K - K;
        else return 0;
    }
}

File: in_Python/0908 Smallest Range I.py
class Solution(object):
    def smallestRangeI(self, A, K):
        """
        :type A: List[int]
        :type K: int
        :rtype: int
        """
        A.sort()
        newlow = A[0] + K
        newhigh = A[-1] - K
        return max(newhigh - newlow, 0)